wicker/core/column_files.py

```diff
@@ -178,55 +186,106 @@ def _write_column(self, column_name: str) -> None:
         self.write_buffers[column_name] = self._get_new_buffer()
 
 
-class ColumnBytesFileCache:
+class ColumnBytesFileReader(abc.ABC):
```

Comment:
@pickles-bread-and-butter you can remove `abc.ABC`, this class is now a perfect cacheless reader!!!

Reply:
Done 🥳